contracts/extensions/package.json

```diff
@@ -48,7 +48,7 @@
     "homepage": "https://github.com/0xProject/0x-monorepo/contracts/extensions/README.md",
     "devDependencies": {
         "@0x/abi-gen": "^3.1.2",
-        "@0x/contract-wrappers": "^10.1.0",
+        "@0x/contract-wrappers": "10.1.0",
```

Comment:
pinning the version to do a mass refactor later. Some packages will be complicated to refactor, and before publishing, we can't specify `11.0.0` in some packages and `10.1.0` in others. So for now all packages will still use `10.1.0`.